### PR TITLE
Add additional information for ipfix input

### LINKED ISSUE #797: Improve IPFix documentation.
improve on the IPFIX documetation on the graylog documentation.

https://docs.graylog.org/en/3.2/pages/integrations/inputs/ipfix_input.html

### PR DESCRIPTION
**Pull request by cobaltclaudia**
closes #797

### CODE REVIEW

File: pages/integrations/inputs/ipfix_input.rst [hunk 1]
```diff
@@ -34,14 +34,11 @@ For a complete list of the elements see `IPFIX Information Elements page <https:
     {
       "element_id": ELEMENT ID NUMBER,
       "name": "NAME OF DEFINITIONt",
-      "data_type": "basiclist"
+      "data_type": "ABSTRACT DATA TYPE"
```
Comment (lingpri) [inline pages/integrations/inputs/ipfix_input.rst:37]: Just an FYI, I have a image of all the datatypes , that are coded in the spec. I'm wondering, if we can add the picture in here. 

@ceruleancee @jan : Please let us know your thoughts on , if we can include the picture below.

https://user-images.githubusercontent.com/6947168/72761092-84615980-3ba0-11ea-823d-f8cf5aba95ed.png

Comment (jalogisch) [reply pages/integrations/inputs/ipfix_input.rst:37]: I would remove the not needed whitespace on the picture - but yes, please include that.

**Review by jalogisch — CHANGES_REQUESTED** [review]

File: pages/integrations/inputs/ipfix_input.rst (hunk 1 above)
Comment (cobaltclaudia) [reply pages/integrations/inputs/ipfix_input.rst:37]: cool, great idea! image has been added. 👍

**Review by jalogisch — APPROVED** [review]
👍

**Comment by lingpri** [comment]
@ceruleancee , is this a good idea, to include this example below from the community , as an example . may be, state in the documentation, that our current implementation does not support basicList or subTemplatemultiList.

{
        "enterprise_number": 21373,
        "information_elements": [
                {
                        "element_id": 4,
                        "name": "mark",
                        "data_type": "unsigned32"
                },
                {
                        "element_id": 6,
                        "name": "conntrack_id",
                        "data_type": "unsigned32"
                }
        ]
}

**Pull request merged**